### ISSUE TITLE
keep folder around

### ISSUE THREAD

**Issue by kunkinkan** [Squirrel/Squirrel.Windows#718]
When installing a new version (not using updates), AppData/MyApp folder is completely emptied. i'd like to specify which subfolders not to delete. Is that possible? (Shared database, need to keep around).

**Comment by omen7288** [comment]
@kunkinkan did you find an answer for this? I'm currently looking into doing something similar - a persistent set of database files that should exist between various versions of the app.

**Comment by omen7288** [comment]
For anyone curious. I ended up saving persistent data outside of the directory (%LocalAppData%\MyAppName-data) that is normally saved to (%LocalAppData%\MyAppName on windows).  I created this directory during --squirrel-firstrun. I originally was going to show dialog and confirmation to remove this auxiliary data if the user wanted, but found out that you have < 15 seconds (which is indicated in the documentation) to show the dialog and remove the directory which is not enough time in my case to guarantee the user has a) read the dialogs and b) given the hardware enough time to remove the data (might be 20+ gigs in our case). During testing, it would kill the dialogs at 15 seconds which is reasonable.

The interesting thing I should note, I'm in electron, the --squirrel-install event did not last longer than a second before the process was killed and relaunched with --squirrel-firstrun, even though I read in a few places that you had 15 seconds to respond to this.  I had to move my logic that creates the directory into --squirrel-firstrun since we do a bunch of things (registry entries, create directories, shortcut etc.)

I have the following versions for context if this is any help:
node: v4.5.0   ---------(I compiled in sqlite3 and at this time had issues on newer versions of node even though node 6.6+ is out)
npm: 3.10.8
electron:  ^1.4.1
